packages/ui/src/components/Modal/Dialog.tsx

```diff
@@ -75,13 +80,15 @@ export const Dialog = ({
   backdropCss,
 }: DialogProps) => {
   const containerRef = useRef(document.createElement('div'))
+  const dialogRef = useRef(document.createElement('dialog'))
```

Comment:
Why do we have to create an element on every render?

```suggestion
  const containerRef = useRef<HTMLElement>(null)
  const dialogRef = useRef<HTMLElement>(null)
```